internal/pkg/template/templates/workloads/partials/cf/vpc-connector.yml

```diff
@@ -24,7 +24,7 @@ EnvironmentSecurityGroupIngressFromServiceSecurityGroup:
 
 VpcConnector:
   Type: AWS::AppRunner::VpcConnector
-{{- if ne .EnvControllerLambda ""}}
+{{- if gt (len (envControllerParams .)) 0}}
```

Comment:
I see, for RDWS , `{{- if ne .EnvControllerLambda ""}}` and `{{- if gt (len (envControllerParams .)) 0}}` are effectively equivalent because both are true iff network placement is private.